content/articles/q1_2019_meetup.md

```diff
@@ -0,0 +1,25 @@
+Title: San Antonio Devs Meetup - Q1 2019
+Date: 02-12-2019
+Tags: meetup, events
+Slug: q1-2019-meetup
+Authors: Daniel Wallace
+Summary: Information about the first San Antonio Devs meetup of 2019!
+Event-start: 2019-02-20 18:30
+Event-duration: 150m
+Event-location: Alamo Post 2 - The American Legion, 3518 Fredericksburg Road, San Antonio, TX 78201
+
+The first meetup of the San Antonio Devs group is coming up!  Please RSVP at the link Below.
+
+<https://www.eventbrite.com/e/sanantonio-devs-meetup-q1-2019-tickets-56443738742>o
+
```

Comment:
Should we mention food will be for purchase and cash only?

Reply:
yes